src/main/kotlin/solve/importer/view/ButtonView.kt

```diff
@@ -0,0 +1,85 @@
+package solve.importer.view
+
+import javafx.geometry.Insets
+import javafx.scene.control.Button
+import solve.importer.ProjectParser.createAlert
+import solve.main.MainController
+import solve.menubar.view.MenuBarView
+import solve.project.model.ProjectFrame
+import tornadofx.*
+
+
+class ButtonView : View() {
+
+    private val mainController: MainController by inject()
+
+    private val directoryPathView: DirectoryPathView by inject()
+
+    private fun cancelAction() {
+        MenuBarView().importer.close()
```

Comment:
Find ``ImporterView`` in that class, I don't think that such field should be public

Reply:
seem it won't work, because ButtonView is called in ImporterView